lib/person/person.rb

```diff
@@ -59,6 +59,24 @@ def breathe_data(after:)
     }
   end
 
+  def simulate_breathe_to_productive(after:, breathe_events:)
```

Comment:
Do we need a new method? Could change `sync_breathe_to_productive` to accept an optional argument `breathe_events` and fetch them if we don't provide them?

Reply:
Hmm. I wanted this version of a dry run (the data dump dry run) to be as separate as possible from the actual import, so don't accidentally do an Atlassian and import some debug data dump.

Reply:
Hrmm. I just worry about the risk of the two methods diverging. Maybe refactor so they both call the same private method to do the bits they do the same?